content/projects/gsoc/2018/simple-pull-request-job-plugin.adoc

```diff
@@ -38,3 +38,6 @@ If the reports are not in a conventional location,
 users could specify the location using the YML file.
 
 link:https://docs.google.com/document/d/1cuC0AvQG3e4GCjIoCwK3J0tcJVAz1eNDKV8d_zXxlO8/edit[More info about the project]
+
+=== Regular meeting Hours
+Every Wednesday 5 PM to 6 PM UTC
```

Comment:
If you want you can try this link and see of it displays your time zone:
https://www.timeanddate.com/worldclock/fixedtime.html?msg=Simple+Pull+Request+Plugin+Meeting+Hours&iso=20180523T17&ah=1